tests/helpers.py

```diff
@@ -0,0 +1,67 @@
+"""Helper functions for the cookiecutter template tests."""
+
+import os
+import pathlib
+import subprocess
+
+DEFAULT_CONFIG = {
+    "github_owner": "test-user",
+    "project_short_description": "description",
+    "project_name": "Cookiecutter Test",
+    "project_slug": "cookiecutter-test",
+}
+
+
+def _generate_package(
+    config: dict[str, str], path: pathlib.Path
+) -> tuple[subprocess.CompletedProcess[str], pathlib.Path]:
+    """
+    Generate a project from the cookiecutter template.
+
+    Arguments:
+    ---------
+    config: dict
+        A dictionary with values for the cookiecutter template,
+        as defined in the cookiecutter.json
+    path: Path
+        Directory to create package in.
```

Comment:
```suggestion
    Parameters
    ----------
    config
        A dictionary with values for the cookiecutter template,
        as defined in the cookiecutter.json
    path
        Directory to create package in.
```

I think this is a NumPy style docstring? If so I believe the arguments section should be titled 'Parameters' and without a colon. Or if a Google style docstring it should be `Args:`. Also as types of arguments are not required for NumPy style docstrings and are already defined in type annotations I would say from a DRY perspective better to not have repeated in docstring.

Reply:
We never decided what docstrings did we? These are what was in the fixture originally; I have no attachment to the NumPy style. 

(It might be nice to consistently use a format that doesn't require all of the `---------` underlines.)

Reply:
Ah sorry I didn't spot this was just porting what is in the code previously! And yeah agree with choosing a style that avoids the `---` underlines as a pain to make sure these stay aligned

Reply:
We can enforce this with https://docs.astral.sh/ruff/faq/#does-ruff-support-numpy-or-google-style-docstrings. Let's make an issue for that.